packages/newsroom-signup/src/NewsroomProfile/index.tsx

```diff
@@ -68,6 +72,48 @@ export class NewsroomProfile extends React.Component<NewsroomProfileProps, Newsr
       <AddRosterMember charter={this.props.charter} updateCharter={this.props.updateCharter} />,
       <CharterQuestions charter={this.props.charter} updateCharter={this.props.updateCharter} />,
       <SignConstitution charter={this.props.charter} updateCharter={this.props.updateCharter} />,
+      <ApplicationSoFarPage
+        charter={{
+          name: "adg",
```

Comment:
I've just merged my PRs so the charter is now saved to the API so I guess this scintillating test charter can be removed :sparkles: